### PR TITLE
Remove Shapeshift

### PR DESCRIPTION
**Pull request by jjeffryes**
Comments out the Shapeshift button until it's working with all of our addresses.

Note: don't merge yet until we have a clear ETA on when Shapeshift may be compatible.

### CODE REVIEW

**Comment by jjeffryes** [comment]
Note: update this to disable the button, but leave it visible, and show a note explaining this feature is coming soon.

**Comment by rmisio** [comment]
Perhaps we want to mention in the note that they could shapeshift into the wallet if they like?

We could also link to the wallet > receive section. I'd need to make a minor tweak to the wallet launcher in the modal manager (I'll do that as part of the PR that integrates the button into there).

**Comment by jjeffryes** [comment]
> We could also link to the wallet > receive section

I think that's a great idea.

**Comment by jjeffryes** [comment]
Note: add language explaining the address formats, and why some external wallets can't be used with the addresses.

**Comment by jjeffryes** [comment]
@rmisio @morebrownies this is ready for review.

**Comment by morebrownies** [comment]
Will check this out in a bit, thanks

File: js/languages/en-US.json [hunk 1]
```diff
@@ -1118,7 +1118,12 @@
       "note2": "Your payment will be held securely in escrow. If any issues arise, you’ll have the ability to open a dispute.",
       "note3": "*The moderator service fee only applies if a dispute is opened.",
       "note4": "Once you have paid, it may take a bit for the interface to update.",
-      "btnConfirmedPay": "Yes, Pay"
+      "btnConfirmedPay": "Yes, Pay",
+      "altcoinNote": "<b>Altcoins</b> will be available when Shapeshift updates to Segregated Witness.",
```
Comment (rmisio) [inline js/languages/en-US.json]: Should we camel-case shapeshift here?

Comment (jjeffryes) [reply js/languages/en-US.json]: Yes. Good catch.

**Comment by morebrownies** [comment]
Love the copy on the payment screen, super helpful @jjeffryes 

Unfortunately I can't test if ShapeShift is working correctly since it's blocked in Washington state. 😢 

screenshot 2017-09-22 11 15 40

**Comment by morebrownies** [comment]
Looks good otherwise 👍

**Comment by rmisio** [comment]
I'm surprised the ShapeShift folks can't give a more usable error screen there... perhaps they have legal reasons not to. Anyhow, that's out of our control.

**Pull request merged**